geom/dPoly.cpp

```diff
@@ -1412,7 +1417,26 @@ void dPoly::writePoly(std::string filename, std::string defaultColor) {
 
   return;
 }
+void dPoly::clearExtraData(){
+	m_boundingBoxTree.clear();
+	m_vertIndexAnno.clear();
+	m_polyIndexAnno.clear();
 
+}
+const boxTree< dRectWithId> & dPoly::getBoundingBoxTree() const{
```

Comment:
One can add a comment saying that this is used to greatly speed up polygon lookup when the number of polygons is large